src/platform/forms-system/src/js/state/helpers.js

```diff
@@ -349,6 +349,53 @@ export function updateSchemaFromUiSchema(
   return currentSchema;
 }
 
+/**
```

Comment:
These changes are also in a smaller PR here https://github.com/department-of-veterans-affairs/vets-website/pull/11617